includes/abstracts/abstract.llms.admin.table.php

```diff
@@ -399,6 +399,28 @@ public function get_search() {
 		return esc_attr( trim( $this->search ) );
 	}
 
+	/**
+	 * Returns an array of CSS class names to use on this table.
+	 * @return array
+	 * @since   [version]
```

Comment:
Gonna nitpick our documentation here. I KNOW I KNOW that our existing codebase doesn't follow the new standards. I do not have interest or capacity to rewrite the docblock for every existing function in the codebase but I want all new code to follow our new standards and we're updating existing methods as we go.

The documented standards remove the `@version` tag for methods placing them only in the file header

In it's place is an additional `@since` tag recording the changes to the methods.

Finally, the `@since` tags are moved above the method's `@param` and `@return` tags.

See docs: https://github.com/gocodebox/lifterlms/blob/master/docs/documentation-standards.md#changelogs and example https://github.com/gocodebox/lifterlms/blob/master/docs/documentation-standards.md#functions-and-class-methods

I know this is hypocritical and nitpicky but I want new code to follow the standards. Thanks for understanding <3

Reply:
I understand completely and I was afraid when I submitted this that I forgot to check on somehting. :)